content/post/how-to-be-bleeding-edge-c++-proposals.md

```diff
@@ -0,0 +1,98 @@
++++
+categories = []
+date = "2015-09-05T23:57:10-07:00"
+description = ""
+keywords = []
+title = "How to be bleeding edge – C++ proposals that'll change the way you code"
+
++++
+
+In this article, I'll go over a few of the most recently implemented C++ features and extensions in GCC and Clang. Some might just change the face of your code forever. At the very least, demonstrating your bleeding edge knowledge in interviews will be sure to get you some points.
+
+Initialized lambda captures
+===========================
+
+This is a C++14 feature that as far as I've been concerned means "capture by move" support. As a stickler for optimization, I hate unnecessary copies. With initialized lambda captures, I can be even more of a stickler:
+
+```
+std::vector<Things> largeVector;
+// fill largeVector with lots of stuff
+auto result = std::async([largeVector{std::move(largeVector)}] {
```

Comment:
Brace initialization was intended to be a universal initialization notation that could be applied anywhere. As such, our convention calls for it for consistency's sake. But other than aesthetics, there's no difference here.